draft-ietf-tls-tls13.md

```diff
@@ -2230,24 +2252,30 @@ mode, the server MUST send Certificate ({{certificate}}) and CertificateVerify
 {:/comment}
 {:br}
 
-In order to accept PSK key establishment, the server sends a
-"pre_shared_key" extension with the selected identity.
+Each PSK MUST be associated with a single Hash algorithm. For PSKs established
```

Comment:
Nit: I think that the "MUST be" here can be simply "is" and a MUST can be added to "For externally established PSKs, the Hash algorithm MUST be set when the PSK is established."